### ISSUE TITLE
run command with sceptre credentials by default

### ISSUE THREAD

**Issue by zaro0508** [Sceptre/sceptre-resolver-cmd#4]
When running AWS commands this resolver requires users to either set AWS env vars for the credentials or pass in the credentials with the `--profile` flag.  It would be nice if this resolver can run the command using the credentials (or profile) that was passed to sceptre by default.  It should also allow the user to override the profile if they wish.

**Comment by lukeplausin** [comment]
Hi @zaro0508,

I did look at doing this myself when I wrote the module. At the time that I wrote the plugin (and still it seems) the Sceptre resolver API does not expose any information about the current environment to the resolver. So it would not be possible to implement this unless we make some changes in the core project.

https://github.com/Sceptre/sceptre/blob/master/sceptre/resolvers/__init__.py#L16
